### PR TITLE
fix(cloud-notifications): Contact group listing not accessible

### PR DESCRIPTION
**Pull request by dmyios**
## Description

Added a new endpoint to list contact groups without ACLs
`GET: /configuration/notifications/contact_groups`

**Fixes** # MON-22543

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04 (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by abuathier — APPROVED** [review]

File: centreon/doc/API/v23.10/Configuration/Notification/FindNotifiableContactGroups.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+get:
+  tags:
+    - Notification
+  summary: "List notifiable contact groups."
+  description: "List notifiable contact groups for Cloud Notifications rules configuration."
```
Comment (cg-tw) [inline centreon/doc/API/v23.10/Configuration/Notification/FindNotifiableContactGroups.yaml:5]:
```suggestion
  description: "List notifiable contact groups for Cloud notification rules configuration."
```

File: centreon/doc/API/v23.10/Configuration/Notification/Schema/NotifiableContactGroups.Listing.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+type: object
+required:
+  - id
+  - name
+properties:
+  id:
+    type: integer
+    description: "Contact group id"
```
Comment (cg-tw) [inline centreon/doc/API/v23.10/Configuration/Notification/Schema/NotifiableContactGroups.Listing.yaml:8]:
```suggestion
    description: "Contact group ID"
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -18287,3 +18287,6 @@ msgstr "Erreur lors de l'ajout d'une commande"
 
 msgid "Error while retrieving a command"
 msgstr "Erreur lors de la récuparation d'une commande"
+
+msgid "Error while retrieving contact groups"
+msgstr "Erreur lors de la récuparation des groupes de contacts"
```
Comment (cg-tw) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18292]:
```suggestion
msgstr "Erreur lors de la récupération des groupes de contacts"
```

**Review by cg-tw — DISMISSED** [review]

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/api/endpoints.ts [hunk 4]
```diff
@@ -19,4 +19,4 @@ export const hostsGroupsEndpoint = `${baseEndpoint}/configuration/hosts/groups`;
 export const serviceGroupsEndpoint = `${baseEndpoint}/configuration/services/groups`;
 export const businessViewsEndpoint = `${baseEndpoint}/bam/configuration/business-views`;
 export const usersEndpoint = `${baseEndpoint}/configuration/users`;
-export const contactsGroupsEndpoint = `${baseEndpoint}/configuration/contacts/groups`;
+export const contactsGroupsEndpoint = `${baseEndpoint}/configuration/notifications/contact_groups`;
```
Comment (bdauria) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/api/endpoints.ts:22]:
```suggestion
export const contactGroupsEndpoint = `${baseEndpoint}/configuration/notifications/contact_groups`;
```

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelCreate.cypress.spec.tsx [hunk 5]
```diff
@@ -21,9 +21,9 @@ import {
   labelSearchContacts
 } from '../../translatedLabels';
 import { panelWidthStorageAtom } from '../../atom';
-import { contactGroupsEndpoint } from '../../../Authentication/api/endpoints';
 import { platformVersionsAtom } from '../../../Main/atoms/platformVersionsAtom';
 import {
+  contactsGroupsEndpoint,
```
Comment (bdauria) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelCreate.cypress.spec.tsx:26]:
```suggestion
  contactGroupsEndpoint,
```

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelCreate.cypress.spec.tsx [hunk 6]
```diff
@@ -91,7 +91,7 @@ const initialize = (): void => {
   cy.interceptAPIRequest({
     alias: 'contactGroupsEndpoint',
     method: Method.GET,
-    path: `${contactGroupsEndpoint}**`,
+    path: `${contactsGroupsEndpoint}**`,
```
Comment (bdauria) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/specs/PanelCreate.cypress.spec.tsx:94]:
```suggestion
    path: `${contactGroupsEndpoint}**`,
```

**Review by cg-tw — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Review by abuathier — APPROVED** [review]

**Comment by dmyios** [comment]
This PR will be merged from release-23.10-next branch

**Pull request closed without merging**